src/User.py

```diff
@@ -0,0 +1,40 @@
+from HttpRequest import httpRequest
+
+
+class User:
+    def __init__(self):
+        self.token = None
+        self._id = None
+        self.firstName = None
+        self.lastName = None
+        self.email = None
+        self.role = None
+        self.classes = None
+        self.picture = None
```

Comment:
Maybe try to store it globally to later use ?

Reply:
Was already done at the bottom of the file, but extracted it with other global variables to a specific file 'utils/globalVars.py'